aspnetcore/toc.yml

```
@@ -846,6 +846,9 @@
         - name: Kestrel overview
           displayName: deploy, publish, server
           uid: fundamentals/servers/kestrel
+        - name: Use HTTP/3 with Kestrel
```

The HTTP/2 entry is just named "HTTP/2".
```suggestion
        - name: HTTP/3
```